### DISCUSSION TITLE
Question about large beta estimates

### DISCUSSION THREAD

**Discussion by XiaoliZhang91** [cvnlab/GLMsingle#109] [Q&A] [answered]
Hello,
We just started trying out this method. With both the demo data from GLMsingle (Matlab example 1 - event related design) as well as our own datasets, we found that the output of beta estimates (i.e. modelmd) can range between -100 to 100 or even larger. Even when we look at certain ROIs like visual areas, the range can be -60 to 60-ish. Considering this output estimates the percent single change, it seems less plausible to see such large values (and negative values too).
Do you have any suggestions on how to interpret or deal with trials with large betas? 
Thank you very much!

**Comment by kendrickkay** [comment, marked as answer]
The method by which units of betas are computed is to take the estimated amplitude and divide by the mean voxel intensity level and multiply by 100.   One possibility is that the voxels for which you are looking at are very low signal intensity to start with (and for those cases it is "easier" to obtain incidental high percent signal change).  For example, outside of the brain, it is very typical to see very large (either positive or negative) percent signal change.  While this is technically correct, it will likely be the case that the betas in this scenario are highly unreliable / unstable.  So, you could confirm for yourself that these large values do not seem to repeat (across repeated trials), and then hence, it can just be deemed to be "noisy/uninformative" voxels.  

Note that it is possible with very high resolution fMRI (e.g. 1 mm or less) to find some voxels with very large (and reliable) percent signal change, like those near very large veins (e.g. up to 10-12 % BOLD signal change).  In addition, note that the length of your experimental conditions (e.g. event-related vs. block design) will also affect the maximum amplitude obtained for a given trial, so you would have to take this into consideration when trying to understand what's going on in a given experiment.

Does that help?  Some further diagnostics on your data could be informative as to which of these possibilities is the case.

**Reply by XiaoliZhang91** [reply]
Thank you very much for fast and detailed reply! 

Our dataset is a simple visual task, and we are trying MVPA methods on difference visual presentation/labels, like svm decoding. We indeed found that voxels that have low raw intensity tend to have wider range of beta estimates. But even after we have defined our visual ROIs independently and selected only voxels with top R2 values (from GLMsingle output), we still found some voxels that occasionally go beyond +- 30 or 40 percent for certain trials. These "high beta" instances are fewer but they exist in our ROI data.

Our concern is that whether it's necessary to additionally exclude voxels/trials with less plausible beta estimates (e.g., exclude voxels that max beyond +-20 percent; though this criteria is arbitrary that we don't know how to decide yet).

In your paper (Prince et al 2022), I wonder if you performed further voxel selection in addition to selection based on R2 and voxel reliability r? (sorry if I have missed this info in the paper.) From user's perspective, do you recommend use (or not use) a voxel selection based on estimated beta magnitude?

Thank you again for your kind help!

**Reply by kendrickkay** [reply]
Based on what you describe, I'm not sure you need to do any special additional exclusion or anything. If the +- 30 or 40 percent that you observe for a given voxel is reliable, great.  If it's not, then that's also fine, since presumably your analysis will take into account trial-to-trial variability. 

From my perspective, I think the main concern should be to determine if anything is "wrong"/"buggy".  As far as I can tell, there is no evidence that anything is "wrong".

Re: Prince 2022, bear in mind that the R2 metric from GLMsingle output is a little tricky (and requires some deep thinking if you want to give a lot of interpretative weight to it).  Simpler would be something like voxel reliability, and that would be totally fine.  So, in summary, I don't think you need to do any voxel selection based on beta magnitude.  If anything, some sort of metric that takes into account reliability would be fine if you wanted to include that.  (E.g. something like "we included voxels that were in the ROI and which had a t-value of at least 3 (for mean-stimulus > 0), or which had a test-retest reliability of at least r > 0.3, or something like that.)

**Reply by XiaoliZhang91** [reply]
Thank you so much for the explanation. We will try voxel reliability measure with our dataset.
